### PR TITLE
Removed Gradient Blocks

### LINKED ISSUE #19: ui-react-starter: Improve the slider/gradient UI
The current slider/gradient rendering is not ideal. Let's separate them out a bit more visually so that the slider doesn't overlay the gradient.

### PR DESCRIPTION
**Pull request by PragatiVerma18**
Fixes #19 
Removed the gradients entirely. Please refer to the screenshot below for more context:
Screenshot 2021-03-18 at 11 23 58 AM

### CODE REVIEW

**Review by ashryanbeats — CHANGES_REQUESTED** [review]
Changes requested from @kerrishotts and me:

1. Remove/hide the upper right-hand value from the sp-sliders (Docs: https://www.adobe.io/photoshop/uxp/uxp/reference-spectrum/User%20Interface/sp-slider/#hiding-the-value)
2. Remove the gradient divs in the js files

**Comment by PragatiVerma18** [comment]
@ashryanbeats I made the changes that you asked for, please look at the video below for more context:




https://user-images.githubusercontent.com/42115530/111731450-82c11d80-8899-11eb-86fd-fc73f001b5f4.mov

**Review by kerrishotts — CHANGES_REQUESTED** [review]
Let's update this to reflect the latest changes made to the starter (mainly functional React & not using `#` for private fields):  https://github.com/AdobeDocs/uxp-photoshop-plugin-samples/tree/main/ui-react-starter

Otherwise, this looks great.

**Comment by PragatiVerma18** [comment]
> Let's update this to reflect the latest changes made to the starter (mainly functional React & not using `#` for private fields): https://github.com/AdobeDocs/uxp-photoshop-plugin-samples/tree/main/ui-react-starter
> 
> Otherwise, this looks great.

Hey @kerrishotts, thanks for the review. I have updated the components as per your comment, please have a look and let me know if any other changes are needed.

File: ui-react-starter/src/components/ColorPicker.jsx [hunk 1]
```diff
@@ -5,67 +5,88 @@ import { WC } from "../components/WC.jsx";
...
+  return (
+    <div className="colorPicker">
+      <div
+        className="color"
+        style={{ backgroundColor: `rgb(${R}, ${G}, ${B})` }}
+      ></div>
+      <WC onInput={updateColor}>
+        <div className="sliderWithInput">
+          <div className="filledRange">
+            <sp-slider ref={_sldR} data-part="R" value={R} min={0} max={255}>
```
Comment (ashryanbeats) [inline ui-react-starter/src/components/ColorPicker.jsx:46]: Looks like we're still missing the `show-value="false"` attributes on the `sp-slider` components.

**Review by ashryanbeats — CHANGES_REQUESTED** [review]

File: ui-react-starter/src/components/ColorPicker.jsx (hunk 1 above)
Comment (PragatiVerma18) [reply ui-react-starter/src/components/ColorPicker.jsx]: Hey @ashryanbeats, just added it to all the `sp-slider` components.

**Pull request merged**